README.md

```diff
@@ -252,6 +254,7 @@ Asynchronous concurrent distributed networking framework for PHP
 
 ### Promises
 
+* [amphp/amp](https://github.com/amphp/amp) - A non-blocking concurrency framework for PHP applications
```

Comment:
I think it's good to have a list of different promise implementations, that's why I listed it here, too.